### PR TITLE
Small improvements of settingtypes.txt

### PR DESCRIPTION
**Pull request by red-001**
see #4340

### CODE REVIEW

File: builtin/settingtypes.txt [hunk 1]
```diff
@@ -662,15 +662,15 @@ ipv6_server (IPv6 server) bool false
 
 [**Advanced]
 
-#    How many blocks are flying in the wire simultaneously per client.
+#    Maximum number of blocks that are simultaneously send to a client.
```
Comment (SmallJoker) [inline builtin/settingtypes.txt]: `sen**t** per client`

File: builtin/settingtypes.txt [hunk 2]
```diff
@@ -662,15 +662,15 @@ ipv6_server (IPv6 server) bool false
...
 max_simultaneous_block_sends_per_client (Maximum simultaneously blocks send per client) int 10
 
-#    How many blocks are flying in the wire simultaneously for the whole server.
-max_simultaneous_block_sends_server_total (Maximum simultaneously bocks send total) int 40
+#   Maximum number of blocks that are simultaneously send.
```
Comment (SmallJoker) [inline builtin/settingtypes.txt]: maybe `sent in total`?

**Comment by SmallJoker** [comment]
It's called `improvements`

**Comment by red-001** [comment]
done

File: builtin/settingtypes.txt [hunk 3]
```diff
@@ -662,15 +662,15 @@ ipv6_server (IPv6 server) bool false
 
 [**Advanced]
 
-#    How many blocks are flying in the wire simultaneously per client.
+#    Maximum number of blocks that are simultaneously send pre client.
```
Comment (paramat) [inline builtin/settingtypes.txt]: 'sent per'

File: builtin/settingtypes.txt [hunk 4]
```diff
@@ -662,15 +662,15 @@ ipv6_server (IPv6 server) bool false
...
 max_simultaneous_block_sends_per_client (Maximum simultaneously blocks send per client) int 10
```
Comment (paramat) [inline builtin/settingtypes.txt]: (Maximum simultaneous block sends per client)

File: builtin/settingtypes.txt [hunk 5]
```diff
@@ -662,15 +662,15 @@ ipv6_server (IPv6 server) bool false
...
 
-#    How many blocks are flying in the wire simultaneously for the whole server.
-max_simultaneous_block_sends_server_total (Maximum simultaneously bocks send total) int 40
+#   Maximum number of blocks that are simultaneously send in total.
```
Comment (paramat) [inline builtin/settingtypes.txt]: 'sent'

File: builtin/settingtypes.txt [hunk 6]
```diff
@@ -662,15 +662,15 @@ ipv6_server (IPv6 server) bool false
...
+max_simultaneous_block_sends_server_total (Maximum simultaneously blocks send total) int 40
```
Comment (paramat) [inline builtin/settingtypes.txt]: (Maximum simultaneous block sends total)

**Comment by paramat** [comment]
Line comments.

**Comment by red-001** [comment]
Updated

File: builtin/settingtypes.txt [hunk 7]
```diff
@@ -662,15 +662,15 @@ ipv6_server (IPv6 server) bool false
 
 [**Advanced]
 
-#    How many blocks are flying in the wire simultaneously per client.
-max_simultaneous_block_sends_per_client (Maximum simultaneously blocks send per client) int 10
+#    Maximum number of blocks that are simultaneously sent per client.
+max_simultaneous_block_sends_per_client (Maximum simultaneously block sends per client) int 10
```
Comment (paramat) [inline builtin/settingtypes.txt]: Remove 'ly' from 'simultaneously'

File: builtin/settingtypes.txt [hunk 8]
```diff
@@ -662,15 +662,15 @@ ipv6_server (IPv6 server) bool false
...
 
-#    How many blocks are flying in the wire simultaneously for the whole server.
-max_simultaneous_block_sends_server_total (Maximum simultaneously bocks send total) int 40
+#   Maximum number of blocks that are simultaneously sent in total.
+max_simultaneous_block_sends_server_total (Maximum simultaneously block sends total) int 40
```
Comment (paramat) [inline builtin/settingtypes.txt]: Remove 'ly' from 'simultaneously'

**Comment by paramat** [comment]
All good except 2 things.

**Comment by paramat** [comment]
:+1:

**Comment by paramat** [comment]
fbe4a9267ff75dc8f27ea1f6bcd59bc6727f347a

**Pull request closed without merging**